brian-ndungu/ongeza_website
# fix(general): investigate issue with prettyPhoto css not loading

**Issue: {{OL-1}}**
Closes #1 

## Type
* Quick fix.

## Completed
* Added missing prettyPhoto css.

## Screenshots
* **Currently on master, seems like prettyPhoto css is missing from the css folder and thus fails to load**
![prettyPhoto_css_missing](https://user-images.githubusercontent.com/30522249/100794452-453d9f00-3426-11eb-90fc-57cc10d88136.png)

* **Proposed fix, added missing css file**
![added-missing-prettyPhoto-css-file](https://user-images.githubusercontent.com/30522249/100794706-9f3e6480-3426-11eb-9ab5-6945387ebc04.gif)


## Verification Process
* Pull the branch.
* Load index.html in the browser and verify that the site loads without any issues.
